index.html

```diff
@@ -381,10 +381,10 @@ <h3>Information for User Agents</h3>
 	      <li>For platforms that expose <a>desktop focus</a> or <a>accessibility <abbr title="Application Programing Interfaces">API</abbr></a> focus separately from <abbr title="Document Object Model">DOM</abbr> focus, do not expose the focused state in the accessibility <abbr title="application programming interface">API</abbr> for any element when it has <abbr title="Document Object Model">DOM</abbr> focus and also has <pref>aria-activedescendant</pref> which points to a valid <a href="#valuetype_idref">ID reference</a>.</li>
 	      <li>When the <pref>aria-activedescendant</pref> attribute changes on an element that currently has <abbr title="Document Object Model">DOM</abbr> focus, remove the focused state from the previously focused object and fire an accessibility <abbr title="application programming interface">API</abbr> <a>desktop focus event</a> on the new element referenced by <code>aria-activedescendant</code>. If <pref>aria-activedescendant</pref> is cleared or does not point to an element in the current document, fire a desktop focus event for the <a>object</a> that had the attribute change.</li>
-	      <li>For any element with an ID attribute, where the element is a descendant of an element with the <pref>aria-activedescendant</pref> attribute, apply the following accessibility <abbr title="Application Programming Interface">API</abbr> states to the target to ensure the object is accessible:
+	      <li>Apply the following accessibility <abbr title="Application Programming Interface">API</abbr> states to any element with an ID attribute that is also either <a>owned</a> by an element with the <pref>aria-activedescendant</pref> property or is <a>owned</a> by an element that is controlled by an element with the <pref>aria-activedescendant</pref> attribute:
 		<ol>
-		  <li>Focusable, if the element also has a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> <a class="termref">role</a>, because the <pref>aria-activedescendant</pref> of the container can potentially point to it. It is not  absolutely necessary to check this when there is no role, because elements that would be focusable would already have the focusable state.</li>
-		  <li>Focused, whenever the container element sets <pref>aria-activedescendant</pref> to match the ID of this descendant and the container widget with <pref>aria-activedescendant</pref> has <abbr title="Document Object Model">DOM</abbr> focus.</li>
+		  <li>Focusable, if the element also has a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> <a class="termref">role</a>. It should be focusable because the element could be referenced by the <pref>aria-activedescendant</pref> property. It is not  absolutely necessary to check this when there is no role because elements that would be focusable would already have the focusable state.</li>
```

Comment:
Removing the word "should" because we are trying not to use lowercase normative words.
```suggestion
		  <li>Focusable, if the element also has a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> <a class="termref">role</a>. The element needs to be focusable because it could be referenced by the <pref>aria-activedescendant</pref> property. It is not  absolutely necessary to check this when there is no role because elements that would be focusable would already have the focusable state.</li>
```